Dockerfile

```diff
@@ -11,13 +11,13 @@ FROM ghcr.io/siderolabs/ca-certificates:v1.7.0 AS image-ca-certificates
 FROM ghcr.io/siderolabs/fhs:v1.7.0 AS image-fhs
 
 # runs markdownlint
-FROM docker.io/node:22.2.0-alpine3.19 AS lint-markdown
+FROM docker.io/oven/bun:1.1.12-alpine AS lint-markdown
 WORKDIR /src
-RUN npm i -g markdownlint-cli@0.41.0
-RUN npm i sentences-per-line@0.2.1
+RUN bun i -g markdownlint-cli@0.41.0
+RUN bun i sentences-per-line@0.2.1
```

Comment:
let's merge in the same line, so we don't run two commands (it supports multiple target packages to install)